otter/worker/_rcv3.py

```diff
@@ -27,6 +28,11 @@ def _generic_rcv3_request(step_class, request_bag, lb_id, server_id):
         request has no body.
     """
     effect = step_class(lb_node_pairs=[(lb_id, server_id)])._bare_effect()
+
+    svc_req = effect.intent
```

Comment:
Thanks.  :)  Appreciate @kivattik is currently adding a test that would fail if 409's were to prevent deletes from happening (the test is otter should still be able to scale down if the user had already removed the server from the load balancer pool)